handbook/python-dataplugins/python-dataplugins.md

```diff
@@ -0,0 +1,308 @@
+# Python DataPlugins
+
+Create a DataPlugin to load, to register, or to search your own file formats in LabVIEW or DIAdem, or to index and to browse your own file formats with SystemLink DataFinder.
+
+You can create DataPlugins using C++, VBS, LabVIEW or Python.

```

Comment:
You can also create DataPlugins using C++, VBS, or LabVIEW. 
Later on, when we know where to point programmers to, we could add: "For more information on how to create DataPlugins with C++, VBS, or G code, visit "http://..." "

We could also add something like: "Browse through 230+ DataPlugins in the NI Store (url) to check whether there is an existing DataPlugin for your data format.

Reply:
Good idea. We could already add your last sentence like this:
![image](https://user-images.githubusercontent.com/79531/104427009-00f40180-5583-11eb-888c-686aac726f5f.png)